From d680dfa359403480810df83987d19b3a7dcb2c7d Mon Sep 17 00:00:00 2001
From: Eric Seibel <eric.seibel@intel.com>
Date: Tue, 24 Sep 2013 15:12:49 +0200
Subject: [PORT FROM MAIN] Rename saltbay_pr1 and cleaning

BZ: 140721 140739

- Rename saltbay_pr1 to saltbay
- Remove obsolete build target

Change-Id: I7ad7e02464451b203914befa1fea4e2c196737d3
Orig-Change-Id: I9cb2be4731913afcd2dd1f5526cad346bcb76e9f
Signed-off-by: Eric Seibel <eric.seibel@intel.com>
Reviewed-on: http://android.intel.com:8080/133967
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Fert, Laurent <laurent.fert@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
Signed-off-by: Xavier Boniface <xavierx.boniface@intel.com>
---
 conf/intel/Android.mk                  |   17 +++++++++++------
 conf/intel/bodegabay/Android.mk        |   10 ----------
 conf/intel/bodegabay/bt_vendor.conf    |    2 --
 conf/intel/byt_t_ffrd10/Android.mk     |   10 ----------
 conf/intel/byt_t_ffrd10/bt_vendor.conf |    5 -----
 conf/intel/ctpscalelt/Android.mk       |   10 ----------
 conf/intel/ctpscalelt/bt_vendor.conf   |    5 -----
 conf/intel/saltbay/Android.mk          |   10 ++++++++++
 conf/intel/saltbay/bt_vendor.conf      |    2 ++
 conf/intel/saltbay_pr0/Android.mk      |   10 ----------
 conf/intel/saltbay_pr0/bt_vendor.conf  |    2 --
 conf/intel/saltbay_pr1/Android.mk      |   10 ----------
 conf/intel/saltbay_pr1/bt_vendor.conf  |    2 --
 conf/intel/victoriabay/Android.mk      |   10 ----------
 conf/intel/victoriabay/bt_vendor.conf  |    2 --
 15 files changed, 23 insertions(+), 84 deletions(-)
 delete mode 100644 conf/intel/bodegabay/Android.mk
 delete mode 100644 conf/intel/bodegabay/bt_vendor.conf
 delete mode 100644 conf/intel/byt_t_ffrd10/Android.mk
 delete mode 100644 conf/intel/byt_t_ffrd10/bt_vendor.conf
 delete mode 100644 conf/intel/ctpscalelt/Android.mk
 delete mode 100644 conf/intel/ctpscalelt/bt_vendor.conf
 create mode 100644 conf/intel/saltbay/Android.mk
 create mode 100644 conf/intel/saltbay/bt_vendor.conf
 delete mode 100644 conf/intel/saltbay_pr0/Android.mk
 delete mode 100644 conf/intel/saltbay_pr0/bt_vendor.conf
 delete mode 100644 conf/intel/saltbay_pr1/Android.mk
 delete mode 100644 conf/intel/saltbay_pr1/bt_vendor.conf
 delete mode 100644 conf/intel/victoriabay/Android.mk
 delete mode 100644 conf/intel/victoriabay/bt_vendor.conf

diff --git a/conf/intel/Android.mk b/conf/intel/Android.mk
index 73860b4..e0bab45 100644
--- a/conf/intel/Android.mk
+++ b/conf/intel/Android.mk
@@ -4,18 +4,23 @@ include $(CLEAR_VARS)
 
 LOCAL_MODULE := bt_vendor.conf
 LOCAL_MODULE_TAGS := optional
+ifeq (, $(filter %_next, $(TARGET_PRODUCT)))
 LOCAL_REQUIRED_MODULES := \
 	bt_baylake.conf \
 	bt_byt_m_crb.conf \
-	bt_byt_t_ffrd10.conf \
 	bt_byt_t_ffrd8.conf \
-	bt_bodegabay.conf \
 	bt_ctp7160.conf \
-	bt_ctpscalelt.conf \
 	bt_redhookbay.conf \
-	bt_saltbay_pr0.conf \
-	bt_saltbay_pr1.conf \
-	bt_victoriabay.conf
+	bt_saltbay.conf
+else
+LOCAL_REQUIRED_MODULES := \
+	bt_baylake_next.conf \
+	bt_byt_m_crb_next.conf \
+	bt_byt_t_ffrd8_next.conf \
+	bt_ctp7160_next.conf \
+	bt_redhookbay_next.conf \
+	bt_saltbay_next.conf
+endif
 
 include $(BUILD_PHONY_PACKAGE)
 
diff --git a/conf/intel/bodegabay/Android.mk b/conf/intel/bodegabay/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/bodegabay/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/bodegabay/bt_vendor.conf b/conf/intel/bodegabay/bt_vendor.conf
deleted file mode 100644
index e6d8154..0000000
--- a/conf/intel/bodegabay/bt_vendor.conf
+++ /dev/null
@@ -1,2 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
diff --git a/conf/intel/byt_t_ffrd10/Android.mk b/conf/intel/byt_t_ffrd10/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/byt_t_ffrd10/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/byt_t_ffrd10/bt_vendor.conf b/conf/intel/byt_t_ffrd10/bt_vendor.conf
deleted file mode 100644
index e23e986..0000000
--- a/conf/intel/byt_t_ffrd10/bt_vendor.conf
+++ /dev/null
@@ -1,5 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/ctpscalelt/Android.mk b/conf/intel/ctpscalelt/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/ctpscalelt/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/ctpscalelt/bt_vendor.conf b/conf/intel/ctpscalelt/bt_vendor.conf
deleted file mode 100644
index e23e986..0000000
--- a/conf/intel/ctpscalelt/bt_vendor.conf
+++ /dev/null
@@ -1,5 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
-
-# Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
diff --git a/conf/intel/saltbay/Android.mk b/conf/intel/saltbay/Android.mk
new file mode 100644
index 0000000..57ceaba
--- /dev/null
+++ b/conf/intel/saltbay/Android.mk
@@ -0,0 +1,10 @@
+LOCAL_PATH:= $(call my-dir)
+include $(CLEAR_VARS)
+
+LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
+$(call print-vars $(LOCAL_MODULE))
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
+LOCAL_SRC_FILES := bt_vendor.conf
+include $(BUILD_PREBUILT)
diff --git a/conf/intel/saltbay/bt_vendor.conf b/conf/intel/saltbay/bt_vendor.conf
new file mode 100644
index 0000000..e6d8154
--- /dev/null
+++ b/conf/intel/saltbay/bt_vendor.conf
@@ -0,0 +1,2 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
diff --git a/conf/intel/saltbay_pr0/Android.mk b/conf/intel/saltbay_pr0/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/saltbay_pr0/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/saltbay_pr0/bt_vendor.conf b/conf/intel/saltbay_pr0/bt_vendor.conf
deleted file mode 100644
index e6d8154..0000000
--- a/conf/intel/saltbay_pr0/bt_vendor.conf
+++ /dev/null
@@ -1,2 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
diff --git a/conf/intel/saltbay_pr1/Android.mk b/conf/intel/saltbay_pr1/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/saltbay_pr1/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/saltbay_pr1/bt_vendor.conf b/conf/intel/saltbay_pr1/bt_vendor.conf
deleted file mode 100644
index e6d8154..0000000
--- a/conf/intel/saltbay_pr1/bt_vendor.conf
+++ /dev/null
@@ -1,2 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
diff --git a/conf/intel/victoriabay/Android.mk b/conf/intel/victoriabay/Android.mk
deleted file mode 100644
index 57ceaba..0000000
--- a/conf/intel/victoriabay/Android.mk
+++ /dev/null
@@ -1,10 +0,0 @@
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_MODULE := bt_$(notdir $(LOCAL_PATH)).conf
-$(call print-vars $(LOCAL_MODULE))
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := ETC
-LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/bluetooth
-LOCAL_SRC_FILES := bt_vendor.conf
-include $(BUILD_PREBUILT)
diff --git a/conf/intel/victoriabay/bt_vendor.conf b/conf/intel/victoriabay/bt_vendor.conf
deleted file mode 100644
index e6d8154..0000000
--- a/conf/intel/victoriabay/bt_vendor.conf
+++ /dev/null
@@ -1,2 +0,0 @@
-# UART device port where Bluetooth controller is attached
-UartPort = /dev/ttyMFD0
-- 
1.7.4.1

